ROADMAP.md

```diff
@@ -1,10 +1,58 @@
 # Kubeflow Roadmap
 
 ## Kubeflow 1.5 Release, Due: February 2022
-* Kubeflow 1.5 timeline is tracked in [#538](https://github.com/kubeflow/community/pull/538)
-* Additional details will be posted in early 1Q'22
+* Kubeflow 1.5 [milestones and timeline](https://github.com/kubeflow/community/pull/538)
+
+### Themes
+* Increase ML pipeline functionality with aggregation and passing of parameters and artifacts between pipeline components
+* Improve model accuracy and reduce overfitting, especially with hyper parameter tuning
+* Simplify operations and optimize utilization (including spot instance use cases for distributed training) 
+* More consistent user experience - UI appearance, features and naming
+* Improved documentation, tutorials and examples
+* Stretch - Support for K8s 1.22 and associated dependencies (cert mgr, istio)
+
```

Comment:
@nakfour I like this idea.    It might be too much detail for this roadmap update, but if you feel strongly I will add it.  @kimwnasptd we should consider this in the Release team.

Reply:
Will create an issue to track the K8s 1.22 version and for what's pending for each WG, as part of the KF 1.5 release https://github.com/kubeflow/manifests/issues/2112#issuecomment-1024642329